kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1319,6 +1319,15 @@ object KyuubiConf {
       .intConf
       .createWithDefault(1000000)
 
+  val ENGINE_FLINK_FETCH_TIMEOUT: OptionalConfigEntry[Long] =
+    buildConf("kyuubi.session.engine.flink.fetch.timeout")
+      .doc("Result fetch timeout for Flink engine. If the timeout is reached, the result " +
+        "fetch would be stopped and the current fetched would be returned. If no data are " +
+        "fetched, a TimeoutException would be thrown.")
+      .version("1.8.0")
+      .timeConf
+      .createOptional
```

Comment:
what's the style for other configurations, use 0 or None to represent inf? let's algin the style

Reply:
I found `kyuubi.operation.query.timeout` uses `None`.

https://github.com/apache/kyuubi/blob/9e3ac23df71302c5b87dab206cdf96f42811035e/kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala#L1851

Plus, I think `0` sometimes denotes an immediate timeout instead of an infinite timeout, thus `None` is a better choice.